src/Refactoring-Core/RBDeprecateMethodRefactoring.class.st

```diff
@@ -93,17 +103,26 @@ RBDeprecateMethodRefactoring >> callDeprecationMethod [
 		in: ''' , SystemVersion current imageVersionString , ''''
 ]
 
+{ #category : #initialization }
+RBDeprecateMethodRefactoring >> deprecateMethod: aSelector in: aClass [
+	"partial initialization"
+	
+	oldSelector := aSelector asSymbol.
+	class := self classObjectFor: aClass
+]
+
 { #category : #initialization }
 RBDeprecateMethodRefactoring >> deprecateMethod: aSelector in: aClass using: newSel [
+
+	self deprecateMethod: aSelector in: aClass.
 	oldSelector := aSelector asSymbol.
-	newSelector := newSel asSymbol.
 	class := self classObjectFor: aClass
```

Comment:
I think here `oldSelector` and `class` declaration should be removed and `newSelector` should stay?

Reply:
Yes I will fix in the next batch!